src/ray/common/function_descriptor.h

```diff
@@ -230,23 +230,26 @@ class CppFunctionDescriptor : public FunctionDescriptorInterface {
 
   virtual size_t Hash() const {
     return std::hash<int>()(ray::FunctionDescriptorType::kCppFunctionDescriptor) ^
-           std::hash<std::string>()(typed_message_->function_name());
+           std::hash<std::string>()(typed_message_->function_name()) ^
```

Comment:
totally optional: I think absl and boost has hash combine implementations where order is taken into account.

Reply:
Do you mean don't modify the Hash function?

Reply:
It is ok and nothing needs to be changed in this PR. But I think using `^` to combine hashes makes (function_name="a", class_name="b") and (function_name="b", class_name="a") to produce the same output, which may not be desirable. absl and boost have utilities so hash(function_name="a", class_name="b") != hash(function_name="b", class_name="a").